File: matthias/Cplusplus/Code/simple.cpp
// multiplying the elements of an array, with an indexing mistake 
#include <iostream>

using std::cin; 
using std::cout; 
using std::endl; 

int main() {
  int a[3] = {1,2,3};
  int sum = 1; 

  for(int i = 0; i <= 3; i++)
    sum *= a[i];

  cout << sum << " = 6" << endl; 

  return 0; 
}

File: matthias/Cplusplus/Code/simple.java
// multiplying the elements of a vector, with an indexing mistake 
class Main {

  public static void main(String argv[]) {
    int a[] = {1,2,3};
    int sum = 0; 

    for(int i = 0; i <= 3; i++)
      sum *= a[i];

    System.out.println(sum + " = 6"); 
  }
}


